pkg/ui/react-app/src/pages/rules/RulesContent.tsx

```diff
@@ -86,6 +86,9 @@ export const RulesContent: FC<RouteComponentProps & RulesContentProps> = ({ resp
                         <td style={{ backgroundColor: '#F5F5F5' }} className="rule_cell">
                           <GraphExpressionLink title="alert" expr={r.name} />
                           <GraphExpressionLink title="expr" expr={r.query} />
+                          <div>
+                            <strong>for:</strong> {formatDuration(r.duration * 1000)}
+                          </div>
```

Comment:
Same here, a conditional check would be nice `r.duration > 0`

Reply:
Thanks for your reviews. I made the suggested changes.